Update index.ts

wildjo opened the pull request:
Added additional logging to startup process.

removed potential bug from message.reactions.cache.get.remove by removing duplicate removal from inside catch statement, added logging.

Added error checking and log to client.login process.

wildjo committed:
Update index.ts

kacesensitive commented on src/index.ts:
```diff
@@ -17,11 +17,23 @@ const client = new Client({
+  textChannel = client.channels.cache.get(channelIDs[0]) as TextChannel;
+
+  // Send startup messages
+  textChannel.send('Ollama Online! Ready to receive messages preceded by ' + trigger);
```
I'm thinking we remove this, i know for my server i'd rather not have the message pop up every time the bot starts up

Perhaps just comment it out so people can easily adjust if they so wish

wildjo replied:
Great suggestion. I've done so and updated the fork.

kacesensitive reviewed (changes requested):
LG @wildjo just one minor suggestion then i'll merge!

wildjo committed:
Update index.ts

Commented out send to Discord channel, updated comments, per maintainer suggestion.

wildjo committed:
Update index.ts
